apps/emqx/etc/emqx.conf

```diff
@@ -1023,15 +1023,41 @@ broker {
   ## Dispatch strategy for shared subscription
   ##
   ## @doc broker.shared_subscription_strategy
-  ## ValueType: random | round_robin | sticky | hash
+  ## ValueType: random | round_robin | sticky | hash | local
   ##   - random: dispatch the message to a random selected subscriber
   ##   - round_robin: select the subscribers in a round-robin manner
+  ##   - local: select the local subscriber otherwise random cluster-wise
   ##   - sticky: always use the last selected subscriber to dispatch,
   ##     until the subscriber disconnects.
   ##   - hash: select the subscribers by the hash of clientIds
   ## Default: round_robin
   shared_subscription_strategy = round_robin
 
+  ## Per-group dispatch strategy for shared subscription
+  ##
+  ## @doc broker.group.$group_name.shared_subscription_strategy
+  ## ValueType: random | round_robin | sticky | hash | local
+  ##   - random: dispatch the message to a random selected subscriber
+  ##   - round_robin: select the subscribers in a round-robin manner
+  ##   - local: select the local subscriber otherwise random cluster-wise
+  ##   - sticky: always use the last selected subscriber to dispatch,
+  ##     until the subscriber disconnects.
+  ##   - hash: select the subscribers by the hash of clientIds
+  ## Default: round_robin
+  group {
+    local_group {
+      shared_subscription_strategy = round_robin
+    }
+
+    sticky_group {
+      shared_subscription_strategy = sticky
+    }
+
+    round_robin_group {
+      shared_subscription_strategy = round_robin
+    }
+  }
```

Comment:
this is the default config. we may need to comment them out ?

Reply:
Apparently, values which are not present in configuration can't be set or changed.

I mean, if I delete these values, I won't be able to `emqx_config:put` them back for tests.